.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: addnab/docker-run-action@v3
```

Comment:
@suleohis this is the important part from the article. Without this, it won't be able to use our docker image.